### DISCUSSION TITLE
Updated database

### DISCUSSION THREAD

**Discussion by lefauconmine** [docyx/pc-part-dataset#4] [General]
Hi! I was wondering if anyone knows if the data is gonna be updated as 2 years have passed. And if not, is there somewhere I can have an updated and complete version if it? 
Thanks all

**Comment by docyx** [comment]
Hey, I would update the data but unfortunately PCPartPicker has implemented captchas that prevent this script from working.

For whatever reason there aren't that many (up-to-date) datasets or APIs that I could find. Sorry!

**Comment by docyx** [comment]
The data has been updated as of the V2 rewrite (#5)
